src/Main.java

```diff
@@ -1,307 +1,466 @@
+        continue;
+      }
+
+      Integer opcaoInput = 0;
 
-    /**
-     * Opção 2 - Listar Sintomas.
-     *
-     * Este método é responsável pelo menu de listar os sintomas das doenças
-     *
-     * @param scanner
-     * @param listaSintomas
-     * @param input
-     */
-    private static void listarSintomas(Scanner scanner, String[] listaSintomas, String input) {
-
-        System.out.println("\nVamos listar os possíveis sintomas reconhecidos. Para isso, selecione uma doença:");
-
-        do {
-            System.out.println("\n--");
-            System.out.println("1) Influenza (H1N1)\n" + "2) Pneumonia\n" + "3) Corona Virus (Covid-19)");
-            System.out.println("--\n");
-            System.out.print("Escolha uma doença para para listar os sitomas ou digite \"V\" para voltar: ");
-            input = scanner.nextLine();
-            input = input.toUpperCase();
-
-            switch (input) {
-                case "1":
-                    System.out.println("\nVocê selecionou: INFLUENZA (H1N1)\n");
-                    System.out.println("Os sintomas são:\n");
-                    // Este método é o responsável por listar os sintomas da doença dado um intervalo de números predefinido
-                    listarSintomasDoenca(listaSintomas, 0, 9);
-                    listarSintomasDoenca(listaSintomas, 19, 21);
-                    break;
-                case "2":
-                    System.out.println("\nVocê selecionou: PNEUMONIA\n");
-                    System.out.println("Os sintomas são:\n");
-                    // Este método é o responsável por listar os sintomas da doença dado um intervalo de números predefinido
-                    listarSintomasDoenca(listaSintomas, 0, 5);
-                    listarSintomasDoenca(listaSintomas, 15, 18);
-                    break;
-                case "3":
-                    System.out.println("\nVocê selecionou: Corona Virus (Covid-19)\n");
-                    System.out.println("Os sintomas são:\n");
-                    // Este método é o responsável por listar os sintomas da doença dado um intervalo de números predefinido
-                    listarSintomasDoenca(listaSintomas, 0, 5);
-                    listarSintomasDoenca(listaSintomas, 10, 14);
-                    listarSintomasDoenca(listaSintomas, 18, 21);
-                    break;
-                case "V":
-                    System.out.println("");
-                    break;
-                default:
-                    System.out.println("\nOpção inválida.");
-                    break;
-            }
-        } while (!input.equals("V"));
+      try {
+        // Conversão do input inforado pelo usuário
+        opcaoInput = Integer.parseInt(input);
+      } catch(NumberFormatException e) {
+        System.out.println("Informe um número válido.");
+        continue;
+      }
+
+      if (opcaoInput > Main.SINTOMAS.length) {
```

Comment:
A verificação da linha `381` permite que o usuário digite 0 ou um número negativo, uma vez que está só verifica se o input é menor que `Main.SINTOMAS.length`. Isso ocasiona um `ArrayIndexOutOfBoundsException` na linha 
`391` - `System.out.println("> Sintoma: " + opcaoInput + " - " + Main.SINTOMAS[index]);` 
onde o sistema tentará acessar um range inválido do array `Main.SINTOMAS`.
Para evitar isso sugiro a seguinte correção:
- Ao invés de verificar o limite máximo do input em relação ao tamanho do array `Main.SINTOMAS` delimitar um intervalo entre 1 (inclusive) e `Main.SINTOMAS.length`.
```suggestion
      if (opcaoInput < 1 || opcaoInput > Main.SINTOMAS.length) {
```